test/ex_machina/ecto_has_many_test.exs

```diff
@@ -7,6 +7,7 @@ defmodule ExMachina.EctoHasManyTest do
     schema "packages" do
       field :description, :string
       has_many :statuses, ExMachina.EctoHasManyTest.PackageStatus
+      has_many :invoices, ExMachina.EctoHasManyTest.Invoice
```

Comment:
Could you also write a test case that makes this explicit?

Like "when a has many association is not loaded, it still saves" or something like that? And then explicitly set `invoices: Ecto.Association.NotLoaded`

That way it's clear what it's testing and we don't accidentally remove it in the future. I think this also means you could remove this from the migration and just set "statuses" to NotLoaded